src/Platform/OrchidPreset.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+
+namespace Orchid\Platform;
+
+
+use Illuminate\Foundation\Console\Presets\Preset;
+
+class OrchidPreset extends Preset
+{
+    const ORCHID_MIX_CONFIG_PATTERN = "/(\/\* Orchid mix config start \*\/.*\/\* Orchid mix config end \*\/)/s";
+
+    /**
+     * Install the preset.
+     *
+     * @return void
+     */
+    public static function install()
+    {
+        static::updatePackages(true);
+        static::updatePackages(false);
```

Comment:
```php
static::updatePackages(true);
static::updatePackages(false);
```

so it should be?

Reply:
It applies both devDependencies and dependencies in their respected steps.